xarray/structure/combine.py

```diff
@@ -338,18 +352,21 @@ def _new_tile_id(single_id_ds_pair):
 
 def _nested_combine(
     datasets,
-    concat_dims,
+    concat_dim,
```

Comment:
can we ever see multiple dims here (as a list)? If so we might have to reconsider the rename, otherwise the condition where we wrap `concat_dim` in a list could just always happen? We do support hashables in most places (but are not really consistent right now), so a tuple dimension name could break things (not sure, though).